code/modules/client/preference/loadout/loadout_uniform.dm

```diff
@@ -61,17 +61,17 @@
 /datum/gear/uniform/skirt/job/viro
 	display_name = "skirt, virologist"
 	path = /obj/item/clothing/under/rank/virologist/skirt
-	allowed_roles = list("Chief Medical Officer","Medical Doctor")
+	allowed_roles = list("Chief Medical Officer","Medical Doctor" , "Virologist")
```

Comment:
I suppose in that case I need to do the same with the chemists since they have their own and aren't reall supposed to be wearing a MD's skirt. Vice versa for the viro having the MD skirt. If they want it they'll have to go get it since they don't have it nearby at round start.